deploy/csi-operator.yaml

```diff
@@ -103,10 +157,193 @@ spec:
   group: snapshot.storage.k8s.io
   names:
     kind: VolumeSnapshotContent
+    listKind: VolumeSnapshotContentList
     plural: volumesnapshotcontents
+    singular: volumesnapshotcontent
   scope: Cluster
-  version: v1alpha1
-
+  subresources:
+    status: {}
+  preserveUnknownFields: false
```

Comment:
this field is supported in kubernetes 1.15+ https://v1-15.docs.kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/, Deployment will fail on 1.14.

Reply:
done